web/src/app/escalation-policies/PolicyServicesQuery.js

```diff
@@ -16,22 +18,26 @@ const query = gql`
   }
 `
 
-export default class PolicyServicesQuery extends PureComponent {
-  static propTypes = {
-    escalationPolicyID: p.string.isRequired,
+function PolicyServicesQuery(props) {
+  const { data, loading, error } = useQuery(query, {
+    variables: { id: props.escalationPolicyID },
+  })
+
+  if (!data && loading) {
+    return <Spinner />
   }
 
-  render() {
-    return (
-      <Query
-        query={query}
-        render={({ data }) => (
-          <PolicyServicesCard
-            services={data.escalationPolicy.assignedTo || []}
-          />
-        )}
-        variables={{ id: this.props.escalationPolicyID }}
-      />
-    )
+  if (error) {
```

Comment:
We'll also need a check like: `if (!data.escalationPolicy) return <ObjectNotFound />` otherwise it will throw an error for a missing policy.